test/it/uk/gov/hmrc/exports/connector/EmailConnectorSpec.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.connector
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import play.api.test.Helpers.{ACCEPTED, BAD_GATEWAY, BAD_REQUEST}
+import uk.gov.hmrc.exports.base.IntegrationTestSpec
+import uk.gov.hmrc.exports.config.AppConfig
+import uk.gov.hmrc.exports.connectors.EmailConnector
+import uk.gov.hmrc.exports.models.emails.EmailParameter.MRN
+import uk.gov.hmrc.exports.models.emails.TemplateId.DMSDOC_NOTIFICATION
+import uk.gov.hmrc.exports.models.emails.{EmailParameters, SendEmailError, SendEmailRequest}
+import uk.gov.hmrc.http.HttpClient
+
+class EmailConnectorSpec extends IntegrationTestSpec {
+
+  implicit val appConfig: AppConfig = inject[AppConfig]
+
+  val connector = new EmailConnector(inject[HttpClient])(appConfig, global)
```

Comment:
You can simply `inject[EmailConnector]`

Reply:
I find this more explicit. I do not have to consider if AppConfig is a singleton or not.